_component_design/choice-tiles.md

```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
 layout: component
 category: Components
 usage: >
-  Choice tiles are an alternative selector pattern to radio buttons and checkboxes that provide a means of both displaying choices more visually and of providing a more detailed description of a choice.
+  A choice tile represents a selection that a user can make on a page. As an alternative to a radio button or checkbox, a choice tile contains an icon that graphically represents the choice and a description that provides more detail about the choice.
 preview-image: preview-images/choice-tile.svg
 status: stable
 resource: true
-last-modified: 2017-11-12
-helix-ui-css: true
-helix-ui-javascript: true
-pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
+last-modified: 2019-01-03
+helix-ui-css: false
+helix-ui-javascript: false
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use choice tiles when you want to use iconography to present a set of choices to the user.
+
+{% endcolumn %}
+
+</div>
 
-### When to use
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-- Use choices tiles when presenting a set of choices that are easily recognized by iconography.
-- Only use choice tiles when two or more selections are available.
+## Best practices
 
-### Best practices
+Consider the following best practices when you construct choice tiles:
 
-- When using a description within a choice tile keep the text brief.
+- Only use choice tiles when there are two or more selections available. Do not provide more than nine choice tiles.
+- Choice tiles do not define specific iconography. Stay within brand guidelines and use iconography that supports the selection contained in a choice tile.
+- For more complex selections, add an optional description to medium or large choice tiles. Due to space constraints, do not add a description to small choice tiles.
+- Do not mix sizes in a set of choice tiles.
+- If you do not add a description, collapse the padding to the bottom-most padding.
+- Limit the description you include on a choice tile to no more than two lines of text.
```

Comment:
limit description to 100 characters so that the text doesn't flow onto more than 2 lines.

Check character count with Ryan.